docs/Configuration.md

```diff
@@ -40,7 +40,6 @@ You can add these configurations into spark-defaults.conf to enable or disable t
 | spark.gluten.sql.columnar.numaBinding                   | Set up NUMABinding, default is false                                                                                                                                                                                                                                                                                                                                                                                                                 | true                                                 |
 | spark.gluten.sql.columnar.coreRange                     | Set up the core range for NUMABinding, only works when numaBinding set to true. <br /> The setting is based on the number of cores in your system. Use 72 cores as an example.                                                                                                                                                                                                                                                                       | 0-17,36-53 &#124;18-35,54-71                         |
 | spark.gluten.sql.native.bloomFilter                     | Enable or Disable native runtime bloom filter.                                                                                                                                                                                                                                                                                                                                                                                                       | true                                                 |
-| spark.gluten.sql.columnar.fallback.policy               | setting different fallback policies. The options are: "operator" -> fallback on each operator if not supported. "stage" -> fallback on each stage if the number of unsupported operator/expressions in that stage >= `spark.gluten.sql.columnar.wholeStage.fallback.threshold` in AQE env . "query" -> fallback whole query if the number of unsupported operator/expressions >= `spark.gluten.sql.columnar.query.fallback.threshold` in non-AQE env | "operator"                                           |
```

Comment:
The initial idea is to define a basic cost model to calculate the cost on fallback, then apply the different policies at query/stage level. The current design is a sort of mixed, and it's confusing, I agree to remove this part and use the two ` threshold ` first.